tempesta_fw/connection.h

```diff
@@ -87,6 +89,7 @@ typedef struct {
 	struct list_head	msg_queue;
 	spinlock_t		msg_qlock;
 	atomic_t		refcnt;
+	TfwTlsContext		*tls;
```

Comment:
I'd say that TLS connection (lets name it `TfwTlsConnection`) inherits `TfwConnection`.